.github/ISSUE_TEMPLATE/kafka_user_topic_request.md

```diff
@@ -0,0 +1,81 @@
+---
+name: "User support: Kafka User/Topic request"
+about: "Template for users to request Kafka Topics and User credentials/certs."
+title: ""
+labels: user-support
+assignees: ""
+---
+
+## Questionnaire
+
+### Kafka Topic
+
+1. **Topic names**:
+   <!--
+   A list of kafka topic names to be requested.
+   -->
+
+2. **Topic partitions**:
+   <!--
+   Number of topic partitions.
+   -->
+
+   Default: 3
```

Comment:
Can you move the `Default: 3` into the hidden portion and add: 

PARTITIONS: 

And do the same for the ones below ?

Reply:
I think it looks cleaner this way

Reply:
tried something else